src/reporters/json.ts

```diff
@@ -219,6 +250,69 @@ function stepInfo(
   };
 }
 
+async function processScreenshot(screenshot: Buffer) {
+  const img = sharp(screenshot, { sequentialRead: true });
+  const { width, height } = await img.metadata();
+  /**
+   * Chop the screenshot image (1280*720) which is the default
+   * viewport size in to 64 equal blocks for a given image
+   * which can be acheived by keeping the division to 8
+   *
+   * Changing division to 16 would yield us 256 smaller blocks, but it is not
+   * optimal for caching in the ES and proved out to be bad, so we stick to 8
+   */
+  const divisions = 8;
+  const blockWidth = width / divisions;
+  const blockHeight = height / divisions;
+  const reference: ScreenshotReference = {
+    width,
+    height,
+    blockWidth,
+    blockHeight,
+    blocks: [],
+  };
+  const blocks: Array<ScreenshotBlob> = [];
+
+  for (let row = 0; row < divisions; row++) {
+    const top = row * blockHeight;
+    for (let col = 0; col < divisions; col++) {
+      const left = col * blockWidth;
+      const buf = await img
+        .extract({ top, left, width: blockWidth, height: blockHeight })
+        .toFormat(sharp.format.webp)
```

Comment:
self note: check the size of individual blocks using different format.

Reply:
webp even though smaller than png by ~20ms average would yield more savings. Will leave it as webp for now. 

Processing 50 images
webp - 14.36 seconds
png - 13.119 seconds

Reply:
Benchmarks script to play with
```
!(async () => {
  console.time('screenshots');
  for (let i = 0; i < 50; i++) {
    await gatherScreenshots(
      'path/to/.synthetics/pid/screenshots'
    );
  }
  console.timeEnd('screenshots');
})();
```